.spin/cmds.py

```diff
@@ -319,6 +319,10 @@ def check_docs(ctx, pytest_args, n_jobs, verbose, *args, **kwargs):
        from the top-level `__init__.py` file.
 
     """  # noqa: E501
+    try:
```

Comment:
Maybe add a comment here that scipy_doctest depends on numpy and therefore can’t be in a requirements file, so we catch that it isn’t installed at runtime. Hopefully that’s enough to prevent someone from “helpfully” re-adding it in the future.